src/utils/normalize_url_port.ts

```diff
@@ -14,7 +14,7 @@ export const normalizeUrlPort = (url: URL): number => {
         return Number(url.port);
     }
 
-    if (mapping.hasOwnProperty(url.protocol)) {
+    if (Object.prototype.hasOwnProperty.call(mapping, url.protocol)) {
```

Comment:
Why so complicated? :sweat_smile: 

Would `if (url.protocol in mapping)` work?

Reply:
yeah, looks like it should. changed it, thanks.